test/core/join_during_creation.js

```diff
@@ -289,6 +291,13 @@ contract('Core::joinDuringCreation', async (accounts) => {
       assert.isOk(
         isProposalSetInitialized,
       );
+
+      const actualRootOriginObservationBlock = await config.mockCore
+        .rootOriginObservationBlockHeight.call();
+      assert.strictEqual(
+        actualRootOriginObservationBlock.eqn(expectedRootOriginObservationBlock),
+        true,
+      );
```

Comment:
Here the boolean values are checked.
If it fails then it will display a message that expected was true and actual value is false.
So adding a message that `rootOriginObservationBlock` value is not expected should be added.